keyboards/1upkeyboards/1upslider8/mcuconf.h

```diff
@@ -0,0 +1,13 @@
+// Copyright 2023 ziptyze
+// SPDX-License-Identifier: GPL-2.0-or-later#pragma once
+
+#pragma once
+
+#include_next <mcuconf.h>
+
+#undef RP_I2C_USE_I2C0
+#undef RP_I2C_USE_I2C1
+#undef RP_ADC_USE_ADC1
+#define RP_I2C_USE_I2C0 FALSE
```

Comment:
`RP_I2C_USE_I2C0` is false by default for board `GENERIC_RP_RP2040` (`platforms/chibios/boards/GENERIC_RP_RP2040/configs/mcuconf.h`)
```suggestion
#undef RP_I2C_USE_I2C1
#undef RP_ADC_USE_ADC1
```